paasta_tools/check_chronos_jobs.py

```diff
@@ -90,6 +90,23 @@ def sensu_event_for_last_run_state(state):
         return pysensu_yelp.Status.OK
 
 
+def filter_to_most_recent_jobs(jobs):
+    merged = {}
```

Comment:
I don't like the name of this variable. It isn't that you are merging jobs or anything, it is just keeping the most recent "version" of the job output, right?